domains/client/domain-operator/src/domain_worker.rs

```diff
@@ -1,32 +1,222 @@
-//! Shared domain worker functions.
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
Is this refactoring necessary for this PR? If not, would it be better into its own PR?

Reply:
Not really necessary for this PR, will do that next time 👍